expand/expand_test.go

```diff
@@ -61,3 +63,40 @@ func TestConfigNils(t *testing.T) {
 		})
 	}
 }
+
+func TestFields(t *testing.T) {
+	tests := []struct {
+		src  string
+		want []string
+	}{
+		{
+			"{1..4}",
+			[]string{"1", "2", "3", "4"},
+		},
+		{
+			"a{1..4}",
+			[]string{"a1", "a2", "a3", "a4"},
+		},
+	}
+	for i, tc := range tests {
+		word := parseWord(t, tc.src)
+		t.Run(fmt.Sprintf("%02d-run-1", i), func(t *testing.T) {
```

Comment:
why not make this a second loop with two iterations? you also don't need to do subtests, really. the test should become quite a bit shorter.